student.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age, classroom, parent_permission, name)
+    super(age, parent_permission, name)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯(ツ)/¯'
+  end
+
+  def can_use_services?
+    @age >= 18 || @parent_permission
+  end
```

Comment:
- [ ] You should not override this method on the student class, Please make sure to remove this method from here.